### ISSUE TITLE
Internal parsed entities in the internal DTD subset of an SVG content  document

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#1354]
I wrote:

> Meanwhile, nothing in EPUB 3.2 prevents authors from defining internal parsed entities in internal DTD subsets. 

In my understanding, this XML document is an SVG content document.
Here `desc` is an internal parsed entity declared in an internal DTD subset.
Internal parsed entities are useful for repeating something many times.

Do  people agree that this is an SVG content document, as specified in EPUB 3.0, 3.0.1, and 
3.2?  And 3.3?

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE svg [
<!ENTITY desc "<desc></desc>">]>
<svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%">
    &desc;
    <g alignment-baseline="baseline"></g>
</svg>
```

**Comment by llemeurfr** [comment]
I agree. There is no access to the Web to resolve that, therefore it is acceptable.

**Comment by murata2makoto** [comment]
If I were a publisher, I might use this trick heavily if a long string is repeated many times.

**Comment by murata2makoto** [comment]
Are we ready to agree that all versions of EPUB 3 allow such documents as SVG content documents and close this issue?

**Comment by iherman** [comment]
This issue was discussed in [a meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-11-06-epub#section3).

- `RESOLVED:  Merge PR #1368 to address outstanding DTD issues, and close GH issues 1369-1373`
<details><summary><i class="differentiate">View the transcript</i></summary>
Wendy Reid: we had resolutions at the F2F, and further discussions on github<br/>
… and came to a happy place<br/>
Matt Garrish: <a href="https://github.com/w3c/epub-specs/pull/1368">https://github.com/w3c/epub-specs/pull/1368</a><br/>
Matt Garrish: where we ended up was…<br/>
… we put in an allowance for a specific set of external identifiers that we have put in an appendix<br/>
… we have SVG and MathML that are allowed to be used in content docs or in separate files<br/>
… and we made a restriction against external entities in the internal DTD subset<br/>
… so it prevents some security issues but eases authoring<br/>
… so we’ll no longer force people to remove SVG DTDs from tool-generated files<br/>
… I’m hoping this is it :)<br/>
Ivan Herman: tech comment<br/>
… in fact, the changes are such that<br/>
… makes possible something that I’m not sure we really use<br/>
… I can define as part of an internal entity something that won’t go out to the network<br/>
… I’m not sure if this feature is in use<br/>
… formal comment<br/>
… there was a formal resolution on the previous version; this PR slightly changes that<br/>
… can we get a formal resolution to merge, and also close a bunch of issues which were examples of the problem?<br/>
Proposed resolution: Merge PR #1368 to address outstanding DTD issues, and close GH issues 1369-1373 (Wendy Reid)<br/>
Garth Conboy: +1<br/>
Matt Garrish: +1<br/>
Ivan Herman: +1<br/>
Charles LaPierre: +1<br/>
Matthew Chan: +1<br/>
Wendy Reid: +1<br/>
Brady Duga: +1<br/>
George Kerscher: +1<br/>
Laura Brady: +1<br/>
Bill Kasdorf: +1<br/>
Ben Schroeter: +1<br/>
Resolution #1: Merge PR #1368 to address outstanding DTD issues, and close GH issues 1369-1373<br/>
</details>

**Comment by iherman** [comment]
@wareid I believe this should be closed, too
